### ISSUE TITLE
Fonts Not Working

### ISSUE THREAD

**Issue by JakeTheFlare** [Montana-Media-Arts/341-work#51]
I'm having trouble with my fonts in my final.  I created a separate folder for the downloaded fonts I choose to use for my webpage, but they don't seem to effect my text when I view my work on a different computer.  Do I have to have them in my css folder, or is there a special kind of code I have to use to link my fonts?

**Comment by marchendrickson** [comment]
I dont think that you can just link fonts how you have tried. I usually pull the .ttf out of their original folders and put them just straight into the fonts folder. Then you must create a new font.css file as well as link the style sheet just under the original style.css link in your html. It should look like:

link rel="stylesheet" type="text/css" href="./style/style.css"/
  link rel="stylesheet" type="text/css" href="./style/font.css"/

Then the font.css should look like how it was discussed in [Week 8 bottom of the page](https://montana-media-arts.github.io/mart341-webDev/modules/week-8/external-fonts/).
